src/main/java/com/conveyal/datatools/manager/controllers/api/ServerController.java

```diff
@@ -420,27 +524,34 @@ private static void validateSecurityGroupId(LoadBalancer loadBalancer, EC2Info e
             if (securityGroupId != null) {
                 // Set security group to the first value found attached to ELB.
                 ec2Info.securityGroupId = securityGroupId;
```

Comment:
Since we're using this in more contexts, is this attempt to set the security group still ok? Just checking.

Reply:
um, I guess it is. Haven't seen any failures because of this.